### ISSUE TITLE
[feature request] [macOS] spoof should turn off, then turn on device after changing MAC address

### ISSUE THREAD

**Issue by sg-s** [feross/spoof#16]
# Problem: 

After randomising MAC address with

```
sudo spoof randomize <device>
```

where the device is a wifi connection, the computer does not re-connect to the last connected wifi. 

# Proposed Solution

turn the wifi card off and on after randomizing MAC.

**Comment by sg-s** [comment]
you can do this using:

```
networksetup -setairportpower en0 off
networksetup -setairportpower en0 on
```

**Comment by feross** [comment]
Great suggestion. Shipped as 1.2.0!

**Comment by sg-s** [comment]
thanks!
